apps.toml

```
@@ -1138,6 +1138,14 @@ state = "working"
 subtags = [ "forum" ]
 url = "https://github.com/YunoHost-Apps/flarum_ynh"
 
+[flohmarkt]
+category = "publishing"
+subtags = [ "ecommerce" ]
+state = "inprogress"
```

considering tests are passing on CI, i'd set it as "working", there's not much reason to set it as inprogress

```suggestion
state = "working"
```